CTRP.R

```diff
@@ -0,0 +1,60 @@
+library(dplyr)
+
+AUC_1 <- read.csv("AUC_1.csv", header = TRUE, stringsAsFactors = FALSE)
+AUC_2 <- read.csv("AUC_2.csv", header = TRUE, stringsAsFactors = FALSE)
+cancercellline <- read.csv("cancer_cell_line_info.csv", header = TRUE, stringsAsFactors = FALSE)
+compound <- read.csv("compound_info.csv", header = TRUE, stringsAsFactors = FALSE)
+experiment <- read.csv("Experiment_info.csv", header = TRUE, stringsAsFactors = FALSE)
+
+colnames(experiment)[colnames(experiment) == "expt_id"] <- c("experiment_id")
+
+AUC <- rbind(AUC_1, AUC_2)
+
+#first, let's join AUC and experiment, by "expriment_id"
+
+AUC_experiment <- inner_join(AUC, experiment, by = "experiment_id")
+
+#then, let's join AUC_experiment to cancercellline, by "master_ccl_id"
+
+AUC_experiment_ccl <- inner_join(AUC_experiment, cancercellline, by = "master_ccl_id")
+
+#finally, join the rest everything by "master_cpd_id"
+
+fulllist <- inner_join(AUC_experiment_ccl, compound, by = "master_cpd_id")
+
+
+#test this dataset: Which cancer type has the lowest AUC values to the compound "vorinostat"?
+
+vorinostat <- subset(fulllist, fulllist$cpd_name == "vorinostat")
+minimum <- min(vorinostat$area_under_curve)
+lowest_AUC <- vorinostat[vorinostat$area_under_curve == minimum,]
+print(lowest_AUC$cancer_type)
+#"upper_aerodigestive_tract"
+
+
+#For 22RV1, which compound has the lowest AUC value?
+
+RV1 <- subset(fulllist, fulllist$ccl_name == "22RV1")
+min(RV1$area_under_curve)
+lowest_AUC_RV1 <- RV1[RV1$area_under_curve == min(RV1$area_under_curve),]
+print(lowest_AUC_RV1$cpd_name)
+#"leptomycin B"
```

Comment:
For some reason, I did the same thing but got an error / no data in my data set! Strange!

Reply:
I think he uploaded an incomplete version of dataset at first, then correct it later, as stated in his email, maybe that's the reason!